src/cryptsetup/cryptsetup-tokens/cryptsetup-token-util.c

```diff
@@ -0,0 +1,47 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "cryptsetup-token-util.h"
+#include "string-util.h"
+
+int crypt_dump_buffer_to_hex_string(
+                const char *buf,
+                size_t buf_size,
+                char **ret_dump_str) {
+
+        int r;
+
+        assert(ret_dump_str);
+
+        for (unsigned int i = 0; i < buf_size; i++) {
```

Comment:
please use size_t instead of "unsigned int". After all `buf_size` is size_t too and we are talking about the size of an object in memory, hence size_t is right